pkgs/applications/networking/sniffers/qtwirediff/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, qtbase
+, qmake
+, qtwayland
+, wrapQtAppsHook
+}:
+
+
+stdenv.mkDerivation {
+  pname = "qtwirediff";
+  version = "unstable-2022-01-15";
+
+  src = fetchFromGitHub {
+    owner = "aaptel";
+    repo = "qtwirediff";
+    rev = "64e163e0bfaf1e6d33379de153af53820c146411";
+    hash = "sha256-tEDUEMRiYyCFJYmD6EB+2ZocCAprDrmmJLX0Aa76djU=";
+  };
+
+  nativeBuildInputs = [
+    qmake
+    wrapQtAppsHook
+  ];
+
+  buildInputs = [
+    qtbase
+    qtwayland
+  ];
+
+  postInstall = ''
+    mkdir $out/bin
+    cp qtwirediff $out/bin
```

Comment:
```suggestion
    install -Dm755 -T qtwirediff $out/bin/qtwirediff
```

is better I think

Reply:
Yes I think you right. I think I can also remove the ```mkdir $out/bin``` when using install like that.